src/pbrt/textures.cpp

```diff
@@ -180,7 +180,7 @@ std::string SpectrumBilerpTexture::ToString() const {
 }
 
 // CheckerboardTexture Function Definitions
-Float Checkerboard(TextureEvalContext ctx, TextureMapping2D map2D,
+PBRT_CPU_GPU Float Checkerboard(TextureEvalContext ctx, TextureMapping2D map2D,
```

Comment:
As much as possible I'd like to have these sorts of changes that clean up declarations but don't change functionality in a separate PR. This is motivated by 2bae563d803507982e18c81dd169e1fc13389bfb, which fixed an inadvertent bug in the first merge of the HIP-RT foundations in 5374fba8701ea19d64fbd2a32666d17cfef671e6 that broke the CUDA build. It's just easier to chase those down if commits are more clearly broken into foundations that (shouldn't) change functionality and changes that are expected to add/change functionality.